### ISSUE TITLE
Packaging HTML files as assets

### ISSUE THREAD

**Issue by sroze** [zeit/ncc#295]
I'm bundling a Lambda function using the `@now/node-server` builder. The function uses the `fs` to render some template (`.html`) files. Because it doesn't explicitly require them in the code, they get ignored and are not bundled within the function. Would it be possible to use the "asset" feature of ncc to package them explicitly with the function?

Thank you very much!

**Comment by guybedford** [comment]
Yes, assets are included when you use the `__dirname` identifier to refer to them relatively, and then the reference will be rewritten.

Could you share how you were trying to load them? We are trying to understand the various usages, as we'd like to extend this support further.

**Comment by fabien** [comment]
I'm having the same issue when loading templates from a directory during runtime.

Easiest solution would be to be able to specify a list of paths that should be included as-is within the built directory (mapped to single-level, hashed directory names for example). So while the various directories to look for templates are known upfront (and can use `__dirname` for specifying them) the actual files are only resolved during runtime.

**Comment by styfle** [comment]
There will soon be a config options for `includeFiles` so you can specify which template files should be part of the lambda.

See this PR in the `@now/node` builder: https://github.com/zeit/now-builders/pull/277
